params/config.go

```diff
@@ -567,7 +566,7 @@ func (c *NodeConfig) updateGenesisConfig() error {
 
 // DefaultStatusChainGenesisBlock returns the StatusChain network genesis block.
 func (c *NodeConfig) DefaultStatusChainGenesisBlock() (*core.Genesis, error) {
-	genesisJSON, err := static.Asset("config/status-chain-genesis.json")
+	genesisJSON, err := ioutil.ReadFile(GetStatusHome() + "config/status-chain-genesis.json")
```

Comment:
We could use `path.Join` here instead.

Reply:
This path doesn't seem to exist in the current PR (the config is still under `static` folder).